storage/redis.go

```diff
@@ -107,49 +123,51 @@ func NewRedisClusterPool(forceReconnect bool, config RedisStorageConfig) redis.U
 		maxActive = config.MaxActive
 	}
 
-	timeout := 5 * time.Second
+	timeout := 5
 
 	if config.Timeout > 0 {
-		timeout = time.Duration(config.Timeout) * time.Second
-	}
-
-	var tlsConfig *tls.Config
-	if config.RedisUseSSL {
-		tlsConfig = &tls.Config{
-			InsecureSkipVerify: config.RedisSSLInsecureSkipVerify,
-		}
+		timeout = config.Timeout
 	}
 
-	var client redis.UniversalClient
-	opts := &redis.UniversalOptions{
+	opts := &model.RedisOptions{
 		MasterName:       config.MasterName,
 		SentinelPassword: config.SentinelPassword,
-		Addrs:            getRedisAddrs(config),
-		DB:               config.Database,
+		Addrs:            getRedisAddrs(*config),
```

Comment:
we are already using `getRedisAddrs` func in the lib

Reply:
you're right, thank you!